src/scripts/core/visualization-container.js

```diff
@@ -74,6 +74,12 @@
             return this.ctx;
         },
 
+        setVisibility: function (visible) {
+            var node = this.layer.node();
+
+            visible ? node.style.display = 'block' : node.style.display = 'none';
```

Comment:
double-checking - will the default display always be block? is there never a case when it's inline/inline-block etc?